Resculpt/Resculptable.cs

```diff
@@ -104,6 +120,14 @@ public sealed class Resculptable : KMonoBehaviour {
 				var button = new KIconButtonMenu.ButtonInfo(icon, text, OnResculpt,
 					PAction.MaxAction, null, null, null, ResculptStrings.RESCULPT_TOOLTIP);
 				Game.Instance?.userMenu?.AddButton(gameObject, button);
+
+				var rotationButton = new KIconButtonMenu.ButtonInfo("action_direction_both",
+																	"Rotate Art",
+																	new System.Action(this.OnRotateClicked),
+																	Action.BuildMenuKeyO,
+																	tooltipText: "Rotates artwork. {Hotkey}");
```

Comment:
This should be in the `ResculptStrings` so it can be translated

Reply:
moved to ResculptStrings